--- Makefile.in	2014-02-06 02:33:28.000000000 +0100
+++ Makefile.in	2014-03-02 11:54:06.038698481 +0100
@@ -74,8 +74,8 @@
 
 @if JIM_DOCS
 install-docs: docs
-	$(INSTALL_DATA_DIR) $(DESTDIR)$(docdir)
-	$(INSTALL_DATA) Tcl.html $(DESTDIR)$(docdir)
+	$(INSTALL_DATA_DIR) $(DESTDIR)@datadir@/doc/jimtcl
+	$(INSTALL_DATA) Tcl.html $(DESTDIR)@datadir@/doc/jimtcl
 @else
 install-docs:
 @endif
@@ -87,7 +87,7 @@
 install: all @TCL_EXTS@ install-exec install-docs
 	$(INSTALL_DATA_DIR) $(DESTDIR)@libdir@/jim
 	$(INSTALL_DATA) $(LIBJIM) $(DESTDIR)@libdir@
-	$(INSTALL_DATA) @srcdir@/README.extensions @C_EXT_SHOBJS@ $(DESTDIR)@libdir@/jim
+	$(INSTALL_DATA) @srcdir@/README.extensions @C_EXT_SHOBJS@ $(DESTDIR)@datadir@/doc/jimtcl
 	for i in tcltest.tcl @TCL_EXTS@; do $(INSTALL_DATA) @srcdir@/$$i $(DESTDIR)@libdir@/jim; done
 	$(INSTALL_DATA_DIR) $(DESTDIR)@includedir@
 	$(INSTALL_DATA) @srcdir@/jim.h @srcdir@/jim-eventloop.h @srcdir@/jim-signal.h \
@@ -104,9 +104,9 @@
 	rm -f $(DESTDIR)@bindir@/$(JIMSH)
 	rm -f $(DESTDIR)@bindir@/build-jim-ext
 	rm -f $(DESTDIR)@libdir@/$(LIBJIM)
-	for i in README.extensions @C_EXT_SHOBJS@ @TCL_EXTS@; do rm -f $(DESTDIR)@libdir@/jim/$$i; done
+	for i in README.extensions @C_EXT_SHOBJS@ @TCL_EXTS@; do rm -f $(DESTDIR)@datadir@/doc/jimtcl/$$i; done
 	rm -f $(DESTDIR)@includedir@/jim*.h
-	rm -f $(DESTDIR)@datadir@/doc/jim/Tcl.html
+	rm -f $(DESTDIR)@datadir@/doc/jimtcl/Tcl.html
 @else
 install install-exec: all
 uninstall:
